src/blinkeffect.js

```diff
@@ -0,0 +1,20 @@
+const blinkEffect = (element) => {
+  let count = 0;
+  const interval = setInterval(() => {
+    if (count % 2 === 0) {
+      // element.style.backgroundColor = "yellow";
```

Comment:
- [ ] Please kindly remove this commented line of code, this will make your codebase clean. **Note: This was not there in the previous review, it would had been pointed out.** ℹ️